vignettes/baselinenowcast.Rmd

```diff
@@ -342,27 +342,74 @@ eventually reported (black line), but does a decent overall job of correcting
 for the right-truncation observed in the red line (the data prior to the
 nowcast).
 
-What we really want is a probabilistic estimate of the nowcasted
-case counts.
-
 
 # Estimate uncertainty
 
 So far, we've demonstrated how to generate a point estimate of a nowcast.
-We would like to generate probabilistic nowcasts, which will require
-estimating the empirical error between the expected observations and the
-actual observations when applying the delay distribution.
-This assumes a negative binomial observation model.
+We would like to generate probabilistic nowcasts.
+To do so, we will use the error between the observations that we have and what
+we would have nowcasted retrospectively. We will assume that the observations
+follow a negative binomial observation model, and independently estimate the
+dispersion in the negative binomial at each delay $d$.
+
+The method used to estimate the uncertainty in the nowcast simulates exactly
+the iterative process, by generating retrospective reporting
+triangles using what would have been available as of the time, and then for
+each reporting triangle, estimating a delay distribution and generating
+a nowcast.
+
+To do this, we will first generate retrospective reporting triangles using the
+latest reporting triangle, and then we will generate nowcasts from the list
+of reporting triangles. To generate the nowcasts, we will have to specify the
+number of reporting triangles we want to use for the estimation of the
+uncertainty, which we will call `n_history_uncertainty`, as well as the number
+of observations we want to use to estimate the delay when we generate the
+nowcast, `n_history_delay`.
+
+Note, we will need to have more than `n_history_uncertainty + n_history_delay`
```

Comment:
This is a lot of text for a getting started IMO. Here you are just manually setting the default argument right? I would try and aim to keep this more minimal and signpost to the verbose explanation in the function docs for people who are interested

Reply:
Ok will edit. Yes exactly. Would you suggest not even specifying the argument, but just using the default? It seems important to explain the restrictions on how many rows/cols are needed in the triangle to do these things, but maybe it is just making it more confusing than only showing the default behavior

Reply:
If it were me I would just use the default and then in a sentence say this is assuming X thing and if you want to learn more look at ? docs